app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/FriendsRequestList.kt

```diff
@@ -41,18 +47,38 @@ fun FriendsRequestList(
     acceptFriend: (UserModel) -> Unit,
     declineFriend: (UserModel) -> Unit
 ) {
-  Text(
-      LocalContext.current.getString(R.string.friend_req),
-      style = MaterialTheme.typography.titleLarge,
-      modifier = Modifier.padding(8.dp).testTag("FriendRequestTitle"))
+  Row(
+      modifier = Modifier.fillMaxWidth(),
+      horizontalArrangement = Arrangement.Center,
+      verticalAlignment = Alignment.CenterVertically) {
+        Text(
+            LocalContext.current.getString(R.string.friend_req),
+            style = MaterialTheme.typography.titleLarge,
+            modifier = Modifier.padding(8.dp).testTag("FriendRequestTitle"),
+        )
+      }
   when {
     isConnected == ConnectionState.OFFLINE -> {
       ConnectionMissing()
     }
     friendRequests.isEmpty() -> {
-      Text(
-          LocalContext.current.getString(R.string.no_friend),
-          Modifier.padding(8.dp).testTag("NoFriendRequest"))
+      Column(
+          modifier = Modifier.fillMaxSize().testTag("EmptyFriendsList"),
+          verticalArrangement = Arrangement.Center,
+          horizontalAlignment = Alignment.CenterHorizontally) {
+            Icon(
+                painterResource(id = R.drawable.no_friends),
+                contentDescription = "No Activities",
+                modifier = Modifier.size(85.dp).testTag("NoFriendsLogo"))
+            Text(
+                text = LocalContext.current.getString(R.string.no_friends_yet),
+                style =
+                    TextStyle(

```

Comment:
Here again the font is unfortunately hard coded with size 20.sp and wont adapted to the screen size, consider changing it to a font from the package MaterialTheme.typography provided by Android